manifests/0000_70_cluster-network-operator_02_rbac.yaml

```diff
@@ -1,16 +1,48 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cluster-network-operator-service-account
+  namespace: openshift-network-operator
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/single-node-developer: "true"
+
+---
+
 kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: default-account-cluster-network-operator
+  name: cluster-network-operator-cluster-role-binding
   annotations:
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/ibm-cloud-managed: "true"
     include.release.openshift.io/single-node-developer: "true"
 subjects:
 - kind: ServiceAccount
-  name: default
+  name: cluster-network-operator-service-account
   namespace: openshift-network-operator
 roleRef:
   kind: ClusterRole
   name: cluster-admin
   apiGroup: rbac.authorization.k8s.io
+
+---
+
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: default-account-cluster-network-operator
+  annotations:
+    release.openshift.io/delete: "true"
```

Comment:
I think we need to create a jira card to track the removal of thisresource in the n+1 release (4.16):
https://github.com/openshift/enhancements/blob/master/dev-guide/cluster-version-operator/dev/object-deletion.md#subsequent-releases-strategy